### PR TITLE
Add Buttercup v0.6.1

### PR DESCRIPTION
**Pull request by sallar**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/buttercup.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'buttercup' do
+  version '0.6.1'
+  sha256 :no_check
```
Comment (miccal) [inline Casks/buttercup.rb]: If the `version` is set, so must the [sha256](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md).

File: Casks/buttercup.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  # github.com/buttercup-pw/buttercup was verified as official when first introduced to the cask
+  url "https://github.com/buttercup-pw/buttercup/releases/download/0.6.1-alpha/Buttercup-0.6.1-mac.zip"
```
Comment (miccal) [inline Casks/buttercup.rb]: Please change this to `url "https://github.com/buttercup-pw/buttercup/releases/download/#{version}-alpha/Buttercup-#{version}-mac.zip"`

Also please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by miccal — CHANGES_REQUESTED** [review]

**Comment by sallar** [comment]
@miccal Fixed the errors! Thanks.

**Comment by miccal** [comment]
Thank you.

**Review by miccal — APPROVED** [review]

**Pull request merged**